crates/turbopack-ecmascript/src/references/async_module.rs

```diff
@@ -124,7 +127,12 @@ impl AsyncModuleScc {
         for scc in &*this.scope.get_scc_children(this.scc).await? {
             // Because we generated SCCs there can be no loops in the children, so calling
             // recursively is fine.
-            if *AsyncModuleScc::new(*scc, this.scope).is_async().await? {
+            if *AsyncModuleScc::new(*scc, this.scope)
+                .resolve()
```

Comment:
maybe we should start documenting why a `.resolve()` is necessary

Reply:
Technically it's not necessary, but it will avoid the resolve task for `is_async` (`is_async` is called with unresolved `self` argument)

Reply:
add added a comment